src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Nullable.cs

```diff
@@ -264,6 +279,32 @@ public void EmitAttribute_Module()
             });
         }
 
+        [Fact]
+        public void EmitAttribute_NetModule()
```

Comment:
I'm surprised. I thought we synthesized the attribute when needed but not found.

Reply:
We're not synthesizing the attribute for netmodules. This is consistent with other attributes such as `IsReadOnlyAttribute` and `IsByRefLikeAttribute`.

---
In reply to: [171080244](https://github.com/dotnet/roslyn/pull/25087#discussion_r171080244) [](ancestors = 171080244)